packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -128,9 +148,11 @@ const PolygonGraph = (props: Props) => {
         setFocusVisible,
     };
 
-    return numSides === "unlimited"
-        ? UnlimitedPolygonGraph(statefulProps)
-        : LimitedPolygonGraph(statefulProps);
+    return numSides === "unlimited" ? (
+        <UnlimitedPolygonGraph {...statefulProps} />
+    ) : (
+        <LimitedPolygonGraph {...statefulProps} />
+    );
```

Comment:
I changed this to render as components this so I could use hooks inside each component separately. Otherwise it gives me a conditional hook error and crashes in the editor when changing between fixed and unlimited polygon graph type.

I prefer this over having a whole bunch of hooks up here in the parent that may or may not apply to the two different components and passing them down.